source/develop/release-management/features/network/networking-api-security-groups.html.md

```diff
@@ -0,0 +1,597 @@
+
+## User workflow
+
+Since the feature is active by default, the intended workflow - from the oVirt
+engine user's perspective - is:
+
+* create an external network on the ovirt-provider-ovn - its default
+*port_security_enabled* value is true - which can be overriden during engine-setup.
+* create VMs attached to this external network - they will inherit the
+*port_security_attribute* from the network, which will result in dropping all
+IP traffic to that VM.
+* the intended traffic will have to be white-listed. Assuming the user is
+interested in allowing ssh traffic, the following ansible playbook should be
+used, which creates one security group - for which a security group rule
+allowing all **egress** traffic is automatically created -, plus one security
+group rule allowing **ingress** tcp traffic meant for port 22:
+
+~~~~~~
+
+- os_security_group:
+    cloud: ovirt
+    state: present
+    name: my_app_default
+    description: allow ssh
+    register: my_app_default_security_group
+
+- os_security_group_rule:
+    cloud: ovirt
+    state: present
+    security_group: "{{ my_app_default_security_group.id }}"
+    direction: ingress
+    protocol: tcp
+    port_range_min: 22
+    port_range_max: 22
+
+~~~~~~
+
+* finally, the user is required to update the ports, indicating which security
+groups apply to it. The following playbook shows how a single port, referenced
+by name, is updated:
+
+~~~~~~
+
+- os_port:
+    cloud: ovirt
+    state: present
+    name: <port_name>
+    security_groups:
+        - "{{ my_app_default_security_group.id }}"
+
+~~~~~~
+
+### Installation/Upgrade
+
+The install time required updates are described in the section [above](#engine-setup).
+
+On upgrades with existing VMs - having attachments to external networks -
+the ports will **not** be updated, and it will be up to the administrator to
+manually activate port security in the existent ports.
+
+Updating the network's port-security-enabled attribute **will not** cascade to
+the existent ports, as defined [in the Networking API](https://developer.openstack.org/api-ref/network/v2/#port-security).
+
+### Engine-Setup
+
+A new ovirt-provider-ovn related question will be added, asking the user if
+port-security is enabled at network level. The default answer will be 'yes'.
+
+This will lead to all the created ports having port-security active, leading to
+all IP traffic being dropped.
+
+## Design feature descrition
+
+### Mandatory security group creation
+
+There are 2 types of required security groups, that are created automatically
+by the ovirt-provider-ovn, when required. They are:
```

Comment:
I'm confused with "created automatically￼ by the ovirt-provider-ovn" and "when required".. Is it the user's call?

Reply:
Hm, no. It is done automatically by the ovirt-provider-ovn,when required. 

As indicated in the bullets below:
**deny-all group**: the **deny-all** security group, provisioned when the first port with the 
**port-security** flag activated is created

**allow-dhcp** group: Creating a subnet on top of an external
network will provision an **allow-dhcp** security group **for that particular
network**

I'll rework this a bit, see if it becomes clearer.